src/test/resources/features/CreateEvent.feature

```diff
@@ -0,0 +1,67 @@
+@CreatingDifferentTypesOfEvents
+Feature: Create events
+  This feature deals with the create events functionality of the application
+
+  Background: I log in to Google
+    Given I am logged in
+    Then I should see the Create Event button
+
+
+  Scenario Outline: Creating multiple events
```

Comment:
You should add a variable to the name of the scenario outline, otherwise, there won't be separate test cases of all rows in your report.